neqo-transport/src/connection/tests/mod.rs

```diff
@@ -151,6 +155,25 @@ pub fn maybe_authenticate(conn: &mut Connection) -> bool {
     false
 }
 
+/// Compute the RTT variance after `n` ACKs or other RTT updates.
+pub fn rttvar_after_n_updates(n: usize, rtt: Duration) -> Duration {
+    assert!(n > 0);
+    let mut rttvar = rtt / 2;
+    for _ in 1..n {
+        rttvar = rttvar * 3 / 4;
+    }
+    rttvar
+}
+
+/// This inserts a PING frame into packets.
+pub struct Ping {}
```

Comment:
Maybe avoid collisions with other potential uses of Ping (like on the Frame enum).

```suggestion
pub struct PingWriter {}
```